src/libraries/sendtohelixhelp.proj

```diff
@@ -157,6 +149,22 @@
     <HelixPreCommand Include="printenv | grep COMPlus" />
   </ItemGroup>
 
+  <!-- 
+    We are hosting the payloads for the WASM/browser on Kestrel in the xharness process.
+    We also run some network (HTTP and HTTPS) tests that connect to this server.
+    In order to set up an HTTPS endpoint we need to install a development SSL certificate.
+  -->
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' and '$(HelixCommand)' == '' and '$(TargetsWindows)' != 'true' and '$(BrowserHost)' != 'windows'">
+    <!-- install SDK so that, we could use `dotnet dev-certs https`. -->
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <InstallCert>dotnet dev-certs https &amp;&amp; </InstallCert>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' and '$(HelixCommand)' == '' and ('$(TargetsWindows)' == 'true' or '$(BrowserHost)' == 'windows')">
+    <!-- install SSL cert via powershell on windows -->
+    <InstallCert>powershell -command &quot;New-SelfSignedCertificate -FriendlyName &#39;ASP.NET Core HTTPS development certificate&#39; -DnsName @(&#39;localhost&#39;) -Subject &#39;CN = localhost&#39; -KeyAlgorithm RSA -KeyLength 2048 -HashAlgorithm sha256 -CertStoreLocation &#39;Cert:\CurrentUser\My&#39; -TextExtension @(&#39;2.5.29.37={text}1.3.6.1.5.5.7.3.1&#39;,&#39;1.3.6.1.4.1.311.84.1.1={hex}02&#39;,&#39;2.5.29.19={text}&#39;) -KeyUsage DigitalSignature,KeyEncipherment&quot; &amp;&amp; </InstallCert>
```

Comment:
Just rename this to `HelixCommandPrefix`, and use when setting `HelixCommand`, as you were doing earlier.